vmdb/app/models/foreman_provision_task.rb

```diff
@@ -0,0 +1,66 @@
+require 'manageiq_foreman'
+class ForemanProvisionTask < MiqRequestTask
```

Comment:
class name should probably be MiqProvisionConfiguredSystemForeman to match the [workflow](https://github.com/ManageIQ/manageiq/pull/1891).